### PR TITLE
[Pages] Update _redirects limit information

### PR DESCRIPTION
**Pull request by KianNH**
References:
- Tweet from Sid
  https://twitter.com/chatsidhartha/status/1511809202829860874

- Quote from Walshy
  > The static redirect limit is now much higher! You can have up to 1,000 static redirects (from /a to /b) and 100 dynamic redirects (from /:id to /path/:id). Note static ones need to go first in the _redirects file

### CODE REVIEW

**Comment by KianNH** [comment]
Closes #4114

**Comment by KianNH** [comment]
We also reference it in https://developers.cloudflare.com/pages/how-to/use-bulk-redirects/

**Comment by KianNH** [comment]
Aaaaand https://developers.cloudflare.com/pages/platform/limits/

I think that's everywhere?

**Comment by KianNH** [comment]
Squashed commits.

File: content/pages/how-to/use-bulk-redirects.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Handle redirects with Bulk Redirects
 
 # Handle redirects with Bulk Redirects
 
-In this tutorial, you will learn how to use [Bulk Redirects (beta)](/rules/bulk-redirects/) to handle redirects that surpasses the 100 redirect rules limit set by Pages. A [`_redirects`](/pages/platform/limits/#redirects) file has a maximum of 100 redirect rules.
+In this tutorial, you will learn how to use [Bulk Redirects (beta)](/rules/bulk-redirects/) to handle redirects that surpasses the 1,100 redirect rules limit set by Pages. A [`_redirects`](/pages/platform/limits/#redirects) file has a maximum of 1,000 static redirects and 100 splat redirects, for a combined total of 1,100 redirects.
```
Comment (WalshyDev) [inline content/pages/how-to/use-bulk-redirects.md]:
```suggestion
In this tutorial, you will learn how to use [Bulk Redirects (beta)](/rules/bulk-redirects/) to handle redirects that surpasses the 1,100 redirect rules limit set by Pages. A [`_redirects`](/pages/platform/limits/#redirects) file has a maximum of 1,000 static redirects and 100 dynamic redirects, for a combined total of 1,100 redirects.
```

File: content/pages/platform/limits.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ You can have an unlimited number of [preview deployments](/pages/platform/previe
 
 ## Redirects
 
-A `_redirects` file can have a maximum of 100 redirect rules. It is recommended to use [Bulk Redirects](/pages/how-to/use-bulk-redirects/) when you have more than 100 redirect rules.
+A `_redirects` file can have a maximum of 1,000 static redirects and 100 splat redirects, for a combined total of 1,100 redirects. It is recommended to use [Bulk Redirects](/pages/how-to/use-bulk-redirects/) when you have a need for more than the `_redirects` file supports.
```
Comment (WalshyDev) [inline content/pages/platform/limits.md]:
```suggestion
A `_redirects` file can have a maximum of 1,000 static redirects and 100 dynamic redirects, for a combined total of 1,100 redirects. It is recommended to use [Bulk Redirects](/pages/how-to/use-bulk-redirects/) when you have a need for more than the `_redirects` file supports.
```

File: content/pages/platform/redirects.md [hunk 3]
```diff
@@ -35,7 +35,13 @@ filename: _redirects
 /products/:code/:name /products?code=:code&name=:name
 ```
 
-A project is limited to 100 total redirects. Each redirect declaration has a 1000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
+A project is limited to 1,000 static redirects and 100 splat redirects, for a combined total of 1,100 redirects. Each redirect declaration has a 1000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
```
Comment (WalshyDev) [inline content/pages/platform/redirects.md]:
```suggestion
A project is limited to 1,000 static redirects and 100 dynamic redirects, for a combined total of 1,100 redirects. Each redirect declaration has a 1000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
```

File: content/pages/platform/redirects.md [hunk 4]
```diff
@@ -35,7 +35,13 @@ filename: _redirects
...
+
+{{<Aside type= "note">}}
+
+Make sure that static redirects are before splat redirects in your `_redirects` file.
```
Comment (WalshyDev) [inline content/pages/platform/redirects.md]:
```suggestion
Make sure that static redirects are before dynamic redirects in your `_redirects` file.
```

**Review by WalshyDev — COMMENTED** [review]
I'd rename splat -> dynamic

File: content/pages/platform/redirects.md [hunk 5]
```diff
@@ -35,7 +35,13 @@ filename: _redirects
 /products/:code/:name /products?code=:code&name=:name
 ```
 
-A project is limited to 100 total redirects. Each redirect declaration has a 1000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
+A project is limited to 1,000 static redirects and 100 dynamic redirects, for a combined total of 1,100 redirects. Each redirect declaration has a 1000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
```
Comment (deadlypants1973) [inline content/pages/platform/redirects.md]:
```suggestion
A project is limited to 1,000 static redirects and 100 dynamic redirects, for a combined total of 1,100 redirects. Each redirect declaration has a 1,000-character limit. Malformed definitions are ignored. If there are multiple redirects for the same `source` path, the topmost redirect is applied.
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**